### ISSUE TITLE
Handling text emphasis

### ISSUE THREAD

**Issue by bond7o6** [passamo9/LaTeX_EnglandWales_Templates#37]
PLACE-HOLDER: to discuss how emphasis for text should be handled both in tex and the end result.

**Comment by Andonome** [comment]
Broadly, the options are:

1. Everything in bold
2. Everything in italics
3. Make this depend on the type of document
4. Chop-and-change because both are required.
5. Underline things, like a 90's student.

**Comment by passamo9** [comment]
2. Everything in italics

I presume this is a separate issues to implementing a .bib?

Because if it is not, then italics are preferred as this covers both emphasis and book titles.

**Comment by bond7o6** [comment]
`.bib` is over at #39.

In the GDPR doc, some terms were bold, not italic, such as controller. Is there some distinction? Key terms vs standard emphasis? Or is this some edge case we can worry about later?

**Comment by passamo9** [comment]
No, you're right. That's the standard way to make an abbreviation:

>General Data Protection Regulation ("**GDPR**").

I was looking at the Skeleton when I made that comment, and the italics are the latin phrase (never used now in E&W since 1998) and in the titles of cases and books (.bib).

So yes, bold is the better emphasis

**Comment by yjyao** [comment]
i think we are mixing some terms here. i see different uses of italics / bold faces:

### description list

> some terms were bold ... such as controller

i think you meant these?
![image](https://user-images.githubusercontent.com/10174018/108177185-c0266580-70b7-11eb-878b-2140aa301527.png)
this definitely looks more like a description list to me, rather than emphasizing text.

### abbreviations

> General Data Protection Regulation ("GDPR")

this is an abbreviation, not regular text emphasis either.

### quotes

i don't know if the "latin phrase" @passamo9 mentioned was this case or not

### book names
> text emphasis

when i see this i think of the case where you try to stress some points or part of a sentence. like "do you _really_ think

**Comment by passamo9** [comment]
>i think we are mixing some terms here

I agree - thank you so much for setting this all out.

>description lists

Yes, that's a great way of describing that, but the list you have is for contracts.

I was referring to a situation in Pleadings: 

![url](https://user-images.githubusercontent.com/69139574/108212890-b8ee6e80-712e-11eb-9c81-682b9170bf7c.jpg)

This may be similar, but it's not exclusively in description lists that the bolding is used.

>quotes

No, latin phrases are increasing a rarity in English law, but when they are used, they're terms of art, not quotes. The example I used _quia timet_ simply means "Because he fears", but that's not a very good English phrase. Only in 2004 did the ancient writ of _mandamus_ become a Mandatory order. But the _quia timet_ is an exception/edge case, and can be ignored for now to little detriment in my opinion (low priority).

>book names

>i assume italics?

Yes, that would be brilliant. Would the use of .bib be possible here?

>text emphasis

Definitely italics.

Thank you so much @yjyao!! I hope my answers were helpful.
